pages/employee/reports.vue

```diff
@@ -71,7 +72,7 @@
         {name: 'Październik 2023', value: 10},
         {name: 'Listopad 2023', value: 11},
     ]
-    const selectedMonth = ref(months.at(-2));
+    const selectedMonth = ref(months.at(-1));
```

Comment:
po co ustawiać listopad, skoro jeszcze jest pusty (bo dalej trwa)?
```suggestion
    const selectedMonth = ref(months.at(-2));
```

Reply:
przy ładowaniu strony, na początku wszędzie zera, a potem zmienia się na wartości i to brzydko wygląda,
a w listopadzie jest zawsze 0 i nie ma takiej zmiany, po za tym mamy aktualnie listopad

Reply:
no ok